### DISCUSSION TITLE
Zepp OS does support custom BLE broadcasting?

### DISCUSSION THREAD

**Discussion by arstagaev** [zepp-health/zeppos-forum#379] [Q&A]
For example I, as developer, can change BLE broadcasting payload?

**Comment by MyoungXUE** [comment]
Sorry, this ability is not supported for the time being. May I ask what you want it to do? I would like to hear more of your thoughts.

**Reply by arstagaev** [reply]
Hi @MyoungXUE, okay, thank you for the answer! 
I want to make indoor tracking with sending during broadcasting some useful payload (like health data with couple custom business case toggles etc). And BLE broadcasting / advertising so cool fit for that🙂.

By the way, Android (Wear OS) Smart Watches can support this feature.

**Reply by MyoungXUE** [reply]
I've discussed this with my colleagues, and we all agree that this feature is impressive. We'd like to invite you for a more in-depth conversation. Would you be interested in joining our Discord server?

Discord Link https://t.zepp.com/t/b6e70

**Reply by arstagaev** [reply]
Awesome, yes, I would interested join. So, I'll prepare my suggestion about BLE advertising and publish it in suggestion section 🤔
